tools/docs/docs_generate.ts

```diff
@@ -153,13 +153,21 @@ for (const manifestPath of manifestFilePaths) {
       ?.filter(
         (declaration): declaration is ExtendedClassDeclaration => declaration.kind === 'class',
       )
-      .forEach((declaration) =>
+      .forEach((declaration) => {
         ['members', 'slots', 'cssProperties', 'events'].forEach((type) =>
           (declaration[type as keyof ExtendedClassDeclaration] as { name: string }[])?.sort(
             (a, b) => a.name.localeCompare(b.name),
           ),
-        ),
-      );
+        );
+        declaration.members.forEach((member) => {
```

Comment:
nice addition :). shouldn't markdown be used? But it works with HTML, I know :-)

Reply:
Unfortunately this did not work with markdown.